src/biguint.rs

```diff
@@ -1500,7 +1500,7 @@ impl Neg for BigUint {
 
     #[inline]
     fn neg(self) -> BigUint {
-        panic!()
+        panic!("unsigned values cannot be negated")
```

Comment:
:open_mouth: I haven't noticed these nonsense `Neg` impls before -- that dates all the way back to `num`'s extraction from the standard library, before Rust 1.0. I'm inclined to remove them for `num-bigint` 0.3, but that's not for this PR...

Reply:
Here's more context on unsigned `Neg`: https://github.com/rust-lang/rust/pull/23945

Reply:
I did wonder why you bothered impl-ing Neg at all for an unsigned type 😂

Reply:
FYI #145 removed it, so you'll need to drop your change to resolve that merge conflict.